2021/04/21 13:47:44 [error] 9#9: *1 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 179.43.169.181, server: k4b208.p.ssafy.io, request: "GET / HTTP/1.1", host: "k4b208.p.ssafy.io"
2021/04/21 13:53:26 [error] 7#7: *3 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET / HTTP/1.1", host: "k4b208.p.ssafy.io"
2021/04/21 13:53:26 [error] 7#7: *4 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET /favicon.ico HTTP/1.1", host: "k4b208.p.ssafy.io", referrer: "https://k4b208.p.ssafy.io/"
2021/04/21 13:53:35 [error] 7#7: *6 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET /login?from=%2Fjenkins HTTP/1.1", host: "k4b208.p.ssafy.io", referrer: "https://k4b208.p.ssafy.io/jenkins"
2021/04/21 13:53:35 [error] 7#7: *7 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET /favicon.ico HTTP/1.1", host: "k4b208.p.ssafy.io", referrer: "https://k4b208.p.ssafy.io/login?from=%2Fjenkins"
2021/04/21 13:54:12 [error] 7#7: *19 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET /login?from=%2Fjenkins HTTP/1.1", host: "k4b208.p.ssafy.io"
2021/04/21 13:54:12 [error] 7#7: *20 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET /favicon.ico HTTP/1.1", host: "k4b208.p.ssafy.io", referrer: "https://k4b208.p.ssafy.io/login?from=%2Fjenkins"
2021/04/21 13:54:20 [error] 7#7: *21 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET /favicon.ico HTTP/1.1", host: "k4b208.p.ssafy.io", referrer: "https://k4b208.p.ssafy.io/v1/swageer-ui.html"
2021/04/21 13:54:26 [error] 7#7: *22 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET /api/swageer-ui.html HTTP/1.1", host: "k4b208.p.ssafy.io"
2021/04/21 13:54:26 [error] 7#7: *24 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET /favicon.ico HTTP/1.1", host: "k4b208.p.ssafy.io", referrer: "https://k4b208.p.ssafy.io/api/swageer-ui.html"
2021/04/21 13:59:45 [error] 9#9: *1 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET /login?from=%2Fjenkins HTTP/1.1", host: "k4b208.p.ssafy.io", referrer: "https://k4b208.p.ssafy.io/jenkins"
2021/04/21 13:59:45 [error] 8#8: *2 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET /favicon.ico HTTP/1.1", host: "k4b208.p.ssafy.io", referrer: "https://k4b208.p.ssafy.io/login?from=%2Fjenkins"
2021/04/21 14:00:09 [error] 9#9: *4 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET /login?from=%2Fjenkins HTTP/1.1", host: "k4b208.p.ssafy.io", referrer: "https://k4b208.p.ssafy.io/jenkins"
2021/04/21 14:00:09 [error] 9#9: *6 rewrite or internal redirection cycle while internally redirecting to "/index.html", client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET /favicon.ico HTTP/1.1", host: "k4b208.p.ssafy.io", referrer: "https://k4b208.p.ssafy.io/login?from=%2Fjenkins"
2021/04/21 14:00:51 [emerg] 1#1: host not found in upstream "jenkins" in /etc/nginx/conf.d/proxy.conf:42
2021/04/21 14:03:37 [emerg] 1#1: host not found in upstream "jenkins" in /etc/nginx/conf.d/proxy.conf:42
2021/04/21 21:19:12 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET /jenkins HTTP/1.1", upstream: "http://172.18.0.5:8080/jenkins", host: "k4b208.p.ssafy.io"
2021/04/21 21:40:51 [error] 6#6: *302 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "POST /jenkins/job/dev-deploy/buildHistory/ajax HTTP/1.1", upstream: "http://172.18.0.5:8080/jenkins/job/dev-deploy/buildHistory/ajax", host: "k4b208.p.ssafy.io", referrer: "https://k4b208.p.ssafy.io/jenkins/job/dev-deploy/"
2021/04/21 21:41:02 [error] 6#6: *302 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "POST /jenkins/job/dev-deploy/buildHistory/ajax HTTP/1.1", upstream: "http://172.18.0.5:8080/jenkins/job/dev-deploy/buildHistory/ajax", host: "k4b208.p.ssafy.io", referrer: "https://k4b208.p.ssafy.io/jenkins/job/dev-deploy/ws/backend/"
2021/04/22 00:34:50 [error] 6#6: *4790 connect() failed (113: Host is unreachable) while connecting to upstream, client: 45.155.205.27, server: k4b208.p.ssafy.io, request: "GET /api/jsonws/invoke HTTP/1.1", upstream: "http://172.18.0.3:8081/api/jsonws/invoke", host: "k4b208.p.ssafy.io", referrer: "http://13.209.85.224:80/api/jsonws/invoke"
2021/04/22 10:19:24 [error] 6#6: *5847 connect() failed (113: Host is unreachable) while connecting to upstream, client: 45.155.205.27, server: k4b208.p.ssafy.io, request: "POST /api/jsonws/invoke HTTP/1.1", upstream: "http://172.18.0.3:8081/api/jsonws/invoke", host: "13.209.85.224:443"
2021/04/22 21:39:57 [error] 6#6: *5958 connect() failed (113: Host is unreachable) while connecting to upstream, client: 45.155.205.27, server: k4b208.p.ssafy.io, request: "GET /api/jsonws/invoke HTTP/1.1", upstream: "http://172.18.0.3:8081/api/jsonws/invoke", host: "k4b208.p.ssafy.io", referrer: "http://13.209.85.224:80/api/jsonws/invoke"
2021/04/28 08:42:08 [error] 9#9: *4 "/etc/nginx/html/index.html" is not found (2: No such file or directory), client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET / HTTP/1.1", host: "porest.kr"
2021/04/28 08:42:09 [error] 9#9: *4 open() "/etc/nginx/html/favicon.ico" failed (2: No such file or directory), client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET /favicon.ico HTTP/1.1", host: "porest.kr", referrer: "https://porest.kr/"
2021/04/28 08:42:09 [error] 9#9: *4 "/etc/nginx/html/index.html" is not found (2: No such file or directory), client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET / HTTP/1.1", host: "porest.kr"
2021/04/28 08:42:12 [error] 9#9: *4 "/etc/nginx/html/index.html" is not found (2: No such file or directory), client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET / HTTP/1.1", host: "porest.kr"
2021/04/28 08:42:22 [error] 9#9: *4 "/etc/nginx/html/index.html" is not found (2: No such file or directory), client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET / HTTP/1.1", host: "porest.kr"
2021/04/28 08:42:23 [error] 9#9: *4 "/etc/nginx/html/index.html" is not found (2: No such file or directory), client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET / HTTP/1.1", host: "porest.kr"
2021/04/28 08:42:32 [error] 9#9: *4 "/etc/nginx/html/index.html" is not found (2: No such file or directory), client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET / HTTP/1.1", host: "porest.kr"
2021/04/28 08:42:47 [error] 9#9: *4 "/etc/nginx/html/index.html" is not found (2: No such file or directory), client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET / HTTP/1.1", host: "porest.kr"
2021/04/28 08:42:56 [error] 9#9: *4 "/etc/nginx/html/index.html" is not found (2: No such file or directory), client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET / HTTP/1.1", host: "porest.kr"
2021/04/28 08:43:17 [error] 9#9: *10 "/etc/nginx/html/index.html" is not found (2: No such file or directory), client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET / HTTP/1.1", host: "k4b208.p.ssafy.io"
2021/04/28 08:45:14 [error] 9#9: *2 "/etc/nginx/html/index.html" is not found (2: No such file or directory), client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET / HTTP/1.1", host: "k4b208.p.ssafy.io"
2021/04/28 08:45:14 [error] 9#9: *2 open() "/etc/nginx/html/favicon.ico" failed (2: No such file or directory), client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET /favicon.ico HTTP/1.1", host: "k4b208.p.ssafy.io", referrer: "https://k4b208.p.ssafy.io/"
2021/04/28 08:45:27 [error] 9#9: *3 "/etc/nginx/html/index.html" is not found (2: No such file or directory), client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET / HTTP/1.1", host: "k4b208.p.ssafy.io"
2021/04/28 08:45:28 [error] 9#9: *3 open() "/etc/nginx/html/favicon.ico" failed (2: No such file or directory), client: 116.126.81.104, server: k4b208.p.ssafy.io, request: "GET /favicon.ico HTTP/1.1", host: "k4b208.p.ssafy.io", referrer: "https://k4b208.p.ssafy.io/"
2021/04/28 08:47:46 [error] 9#9: *6 "/etc/nginx/html/index.html" is not found (2: No such file or directory), client: 205.169.39.14, server: k4b208.p.ssafy.io, request: "GET / HTTP/1.1", host: "www.porest.kr"
2021/04/28 08:48:13 [error] 9#9: *8 "/etc/nginx/html/index.html" is not found (2: No such file or directory), client: 205.169.39.83, server: k4b208.p.ssafy.io, request: "GET / HTTP/1.1", host: "www.porest.kr"
2021/04/28 08:48:16 [error] 9#9: *10 "/etc/nginx/html/index.html" is not found (2: No such file or directory), client: 205.169.39.117, server: k4b208.p.ssafy.io, request: "GET / HTTP/1.1", host: "www.porest.kr"
2021/04/30 06:22:14 [error] 9#9: *2 open() "/etc/apache2/.htpasswd" failed (2: No such file or directory), client: 116.126.81.104, server: porest.kr, request: "GET / HTTP/1.1", host: "porest.kr"
2021/04/30 06:22:14 [error] 9#9: *2 open() "/etc/apache2/.htpasswd" failed (2: No such file or directory), client: 116.126.81.104, server: porest.kr, request: "GET /favicon.ico HTTP/1.1", host: "porest.kr", referrer: "https://porest.kr/"
2021/04/30 06:22:18 [error] 9#9: *2 open() "/etc/apache2/.htpasswd" failed (2: No such file or directory), client: 116.126.81.104, server: porest.kr, request: "GET / HTTP/1.1", host: "porest.kr"
2021/04/30 06:22:18 [error] 9#9: *2 open() "/etc/apache2/.htpasswd" failed (2: No such file or directory), client: 116.126.81.104, server: porest.kr, request: "GET /favicon.ico HTTP/1.1", host: "porest.kr", referrer: "https://porest.kr/"
2021/04/30 06:22:19 [error] 9#9: *2 open() "/etc/apache2/.htpasswd" failed (2: No such file or directory), client: 116.126.81.104, server: porest.kr, request: "GET / HTTP/1.1", host: "porest.kr"
2021/04/30 06:22:19 [error] 9#9: *2 open() "/etc/apache2/.htpasswd" failed (2: No such file or directory), client: 116.126.81.104, server: porest.kr, request: "GET /favicon.ico HTTP/1.1", host: "porest.kr", referrer: "https://porest.kr/"
2021/04/30 06:22:21 [error] 9#9: *2 open() "/etc/apache2/.htpasswd" failed (2: No such file or directory), client: 116.126.81.104, server: porest.kr, request: "GET / HTTP/1.1", host: "porest.kr"
2021/04/30 06:22:21 [error] 9#9: *2 open() "/etc/apache2/.htpasswd" failed (2: No such file or directory), client: 116.126.81.104, server: porest.kr, request: "GET /favicon.ico HTTP/1.1", host: "porest.kr", referrer: "https://porest.kr/"
2021/04/30 06:22:23 [error] 9#9: *2 open() "/etc/apache2/.htpasswd" failed (2: No such file or directory), client: 116.126.81.104, server: porest.kr, request: "GET / HTTP/1.1", host: "porest.kr"
2021/04/30 06:22:23 [error] 9#9: *2 open() "/etc/apache2/.htpasswd" failed (2: No such file or directory), client: 116.126.81.104, server: porest.kr, request: "GET /favicon.ico HTTP/1.1", host: "porest.kr", referrer: "https://porest.kr/"
2021/04/30 06:22:52 [error] 9#9: *2 open() "/etc/apache2/.htpasswd" failed (2: No such file or directory), client: 116.126.81.104, server: porest.kr, request: "GET /favicon.ico HTTP/1.1", host: "porest.kr", referrer: "https://porest.kr/jenkins/"
2021/04/30 06:36:46 [emerg] 1#1: invalid number of arguments in "auth_basic" directive in /etc/nginx/conf.d/proxy.conf:48
2021/04/30 06:37:54 [emerg] 1#1: invalid number of arguments in "auth_basic" directive in /etc/nginx/conf.d/proxy.conf:48
2021/04/30 06:41:43 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "GET /jenkins/ HTTP/1.1", upstream: "http://172.18.0.2:8080/jenkins/", host: "porest.kr"
2021/04/30 06:46:04 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "GET /jenkins/ HTTP/1.1", upstream: "http://172.18.0.4:8080/jenkins/", host: "porest.kr"
2021/05/01 16:02:03 [error] 11#11: *248 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "POST /jenkins/ajaxBuildQueue HTTP/1.1", upstream: "http://172.18.0.5:8080/jenkins/ajaxBuildQueue", host: "porest.kr", referrer: "https://porest.kr/jenkins/systemInfo"
2021/05/01 16:02:03 [error] 10#10: *546 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "POST /jenkins/ajaxExecutors HTTP/1.1", upstream: "http://172.18.0.5:8080/jenkins/ajaxExecutors", host: "porest.kr", referrer: "https://porest.kr/jenkins/systemInfo"
2021/05/01 16:02:04 [error] 10#10: *607 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "POST /jenkins/ajaxBuildQueue HTTP/1.1", upstream: "http://172.18.0.5:8080/jenkins/ajaxBuildQueue", host: "porest.kr", referrer: "https://porest.kr/jenkins/systemInfo"
2021/05/01 16:02:04 [error] 10#10: *608 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "POST /jenkins/ajaxExecutors HTTP/1.1", upstream: "http://172.18.0.5:8080/jenkins/ajaxExecutors", host: "porest.kr", referrer: "https://porest.kr/jenkins/systemInfo"
2021/05/01 16:13:17 [error] 8#8: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "POST /jenkins/ajaxBuildQueue HTTP/1.1", upstream: "http://172.18.0.5:8080/jenkins/ajaxBuildQueue", host: "porest.kr", referrer: "https://porest.kr/jenkins/systemInfo"
2021/05/01 16:13:17 [error] 8#8: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "POST /jenkins/ajaxExecutors HTTP/1.1", upstream: "http://172.18.0.5:8080/jenkins/ajaxExecutors", host: "porest.kr", referrer: "https://porest.kr/jenkins/systemInfo"
2021/05/01 16:38:42 [error] 7#7: *1959 connect() failed (113: Host is unreachable) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "GET /api/swagger-ui.html HTTP/1.1", upstream: "http://172.18.0.4:8081/api/swagger-ui.html", host: "porest.kr"
2021/05/01 16:41:38 [error] 7#7: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "POST /jenkins/ajaxBuildQueue HTTP/1.1", upstream: "http://172.19.0.3:8080/jenkins/ajaxBuildQueue", host: "porest.kr", referrer: "https://porest.kr/jenkins/systemInfo"
2021/05/01 16:41:38 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "POST /jenkins/ajaxExecutors HTTP/1.1", upstream: "http://172.19.0.3:8080/jenkins/ajaxExecutors", host: "porest.kr", referrer: "https://porest.kr/jenkins/systemInfo"
2021/05/09 07:05:06 [error] 8#8: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "POST /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "https://porest.kr/commingsoon"
2021/05/09 16:39:49 [error] 8#8: *215 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/email-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/email-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/09 16:39:49 [error] 8#8: *211 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/email-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/email-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/09 16:39:50 [error] 8#8: *213 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/email-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/email-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/09 16:54:59 [error] 8#8: *254 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/email-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/email-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/09 16:55:05 [error] 8#8: *250 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/email-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/email-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/09 16:55:05 [error] 8#8: *252 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/email-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/email-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/09 16:55:13 [error] 8#8: *256 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/email-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/email-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/09 16:55:21 [error] 8#8: *258 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/email-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/email-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/09 16:55:23 [error] 8#8: *260 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/email-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/email-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/10 06:23:03 [error] 8#8: *995 connect() failed (111: Connection refused) while connecting to upstream, client: 125.138.75.119, server: porest.kr, request: "GET /api/webjars/springfox-swagger-ui/fonts/titillium-web-v6-latin-regular.woff2 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/webjars/springfox-swagger-ui/fonts/titillium-web-v6-latin-regular.woff2", host: "porest.kr", referrer: "https://porest.kr/api/webjars/springfox-swagger-ui/springfox.css?v=2.9.2"
2021/05/10 06:23:03 [error] 8#8: *995 connect() failed (111: Connection refused) while connecting to upstream, client: 125.138.75.119, server: porest.kr, request: "GET /api/webjars/springfox-swagger-ui/fonts/titillium-web-v6-latin-regular.woff HTTP/1.1", upstream: "http://172.19.0.4:8081/api/webjars/springfox-swagger-ui/fonts/titillium-web-v6-latin-regular.woff", host: "porest.kr", referrer: "https://porest.kr/api/webjars/springfox-swagger-ui/springfox.css?v=2.9.2"
2021/05/12 13:25:00 [error] 8#8: *3519 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "GET /api/swagger-ui.html HTTP/1.1", upstream: "http://172.19.0.4:8081/api/swagger-ui.html", host: "porest.kr"
2021/05/13 14:23:48 [error] 8#8: *4776 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "OPTIONS /api/stories/6/counsels HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/6/counsels", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/14 00:50:47 [error] 8#8: *6688 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/emotions?month=5&year=2021 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/emotions?month=5&year=2021", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/14 00:50:48 [error] 8#8: *6649 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/emotions/1 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/emotions/1", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/14 00:50:49 [error] 8#8: *6680 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/emotions/1 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/emotions/1", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/14 00:50:50 [error] 8#8: *6682 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/emotions/1 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/emotions/1", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/14 00:50:50 [error] 8#8: *6684 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/emotions/6 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/emotions/6", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/14 04:38:19 [error] 8#8: *7206 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "GET /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "https://porest.kr/mailbox/received"
2021/05/16 07:13:48 [error] 8#8: *13804 connect() failed (111: Connection refused) while connecting to upstream, client: 222.235.245.13, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/16 07:13:49 [error] 8#8: *13806 connect() failed (111: Connection refused) while connecting to upstream, client: 222.235.245.13, server: porest.kr, request: "OPTIONS /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/16 07:13:51 [error] 8#8: *13790 connect() failed (111: Connection refused) while connecting to upstream, client: 222.235.245.13, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/16 07:13:52 [error] 8#8: *13785 connect() failed (111: Connection refused) while connecting to upstream, client: 222.235.245.13, server: porest.kr, request: "OPTIONS /api/signup HTTP/1.1", upstream: "http://172.19.0.4:8081/api/signup", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/16 07:13:52 [error] 8#8: *13802 connect() failed (111: Connection refused) while connecting to upstream, client: 222.235.245.13, server: porest.kr, request: "OPTIONS /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/16 07:13:56 [error] 8#8: *13794 connect() failed (111: Connection refused) while connecting to upstream, client: 222.235.245.13, server: porest.kr, request: "OPTIONS /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/16 07:13:57 [error] 8#8: *13796 connect() failed (111: Connection refused) while connecting to upstream, client: 222.235.245.13, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/16 15:30:51 [error] 8#8: *16053 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "GET /api/emotions?month=5&year=2021 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/emotions?month=5&year=2021", host: "porest.kr", referrer: "https://porest.kr/feeling-record"
2021/05/16 17:18:50 [error] 8#8: *16096 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/emotions?month=5&year=2021 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/emotions?month=5&year=2021", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/17 13:03:29 [error] 8#8: *19970 connect() failed (111: Connection refused) while connecting to upstream, client: 125.138.75.119, server: porest.kr, request: "GET /api/emotions?month=5&year=2021 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/emotions?month=5&year=2021", host: "porest.kr", referrer: "https://porest.kr/feeling-record"
2021/05/17 13:03:31 [error] 8#8: *19970 connect() failed (111: Connection refused) while connecting to upstream, client: 125.138.75.119, server: porest.kr, request: "GET /api/emotions?month=6&year=2021 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/emotions?month=6&year=2021", host: "porest.kr", referrer: "https://porest.kr/feeling-record"
2021/05/17 13:03:33 [error] 8#8: *19970 connect() failed (111: Connection refused) while connecting to upstream, client: 125.138.75.119, server: porest.kr, request: "GET /api/emotions?month=5&year=2021 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/emotions?month=5&year=2021", host: "porest.kr", referrer: "https://porest.kr/feeling-record"
2021/05/17 13:03:34 [error] 8#8: *19970 connect() failed (111: Connection refused) while connecting to upstream, client: 125.138.75.119, server: porest.kr, request: "GET /api/emotions?month=6&year=2021 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/emotions?month=6&year=2021", host: "porest.kr", referrer: "https://porest.kr/feeling-record"
2021/05/17 13:03:35 [error] 8#8: *19970 connect() failed (111: Connection refused) while connecting to upstream, client: 125.138.75.119, server: porest.kr, request: "GET /api/emotions?month=5&year=2021 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/emotions?month=5&year=2021", host: "porest.kr", referrer: "https://porest.kr/feeling-record"
2021/05/17 13:03:36 [error] 8#8: *19970 connect() failed (111: Connection refused) while connecting to upstream, client: 125.138.75.119, server: porest.kr, request: "GET /api/emotions?month=4&year=2021 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/emotions?month=4&year=2021", host: "porest.kr", referrer: "https://porest.kr/feeling-record"
2021/05/17 13:03:36 [error] 8#8: *19970 connect() failed (111: Connection refused) while connecting to upstream, client: 125.138.75.119, server: porest.kr, request: "GET /api/emotions?month=3&year=2021 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/emotions?month=3&year=2021", host: "porest.kr", referrer: "https://porest.kr/feeling-record"
2021/05/18 17:16:13 [error] 8#8: *23517 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 19:14:20 [error] 8#8: *24237 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 19:14:20 [error] 8#8: *24239 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 19:14:20 [error] 8#8: *24241 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 19:14:20 [error] 8#8: *24243 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 19:14:20 [error] 8#8: *24245 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 19:14:23 [error] 8#8: *24247 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 19:14:23 [error] 8#8: *24248 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 19:14:23 [error] 8#8: *24249 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 19:14:23 [error] 8#8: *24250 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 19:14:23 [error] 8#8: *24227 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 19:14:26 [error] 8#8: *24255 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 19:14:31 [error] 8#8: *24257 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 19:14:35 [error] 8#8: *24259 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 20:21:54 [error] 9#9: *25163 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/counsels HTTP/1.1", upstream: "http://172.19.0.4:8081/api/counsels", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/18 20:21:54 [error] 8#8: *25162 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/18 20:21:59 [error] 8#8: *25166 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 20:21:59 [error] 8#8: *25168 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/counsels HTTP/1.1", upstream: "http://172.19.0.4:8081/api/counsels", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 20:22:01 [error] 8#8: *25170 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 20:22:01 [error] 8#8: *25158 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/counsels HTTP/1.1", upstream: "http://172.19.0.4:8081/api/counsels", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/18 20:22:01 [error] 8#8: *25159 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/18 20:22:01 [error] 8#8: *25172 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 20:22:02 [error] 8#8: *25174 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/counsels HTTP/1.1", upstream: "http://172.19.0.4:8081/api/counsels", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 20:22:02 [error] 8#8: *25176 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 20:22:03 [error] 8#8: *25178 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 20:22:04 [error] 8#8: *25180 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 20:22:04 [error] 8#8: *25182 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/18 20:22:04 [error] 8#8: *25184 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8081/"
2021/05/19 14:49:10 [error] 8#8: *26858 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "POST /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "https://porest.kr/log/login"
2021/05/19 14:49:15 [error] 8#8: *26858 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "POST /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "https://porest.kr/log/login"
2021/05/19 14:49:17 [error] 9#9: *26854 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "POST /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "https://porest.kr/log/login"
2021/05/19 14:49:19 [error] 9#9: *26854 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "POST /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "https://porest.kr/log/login"
2021/05/19 21:04:03 [error] 8#8: *27922 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/4 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/4", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:04:03 [error] 8#8: *27924 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/0 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/0", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:04:03 [error] 8#8: *27926 connect() failed (113: Host is unreachable) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "OPTIONS /api/email-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/email-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:04:03 [error] 8#8: *27928 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/5 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/5", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:04:06 [error] 8#8: *27930 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/4 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/4", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:04:06 [error] 8#8: *27920 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/5 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/5", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:04:10 [error] 8#8: *27932 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "OPTIONS /api/email-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/email-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:15:19 [error] 8#8: *28031 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/0 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/0", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:15:19 [error] 8#8: *28032 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/0 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/0", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:15:19 [error] 8#8: *28055 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/0 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/0", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:15:19 [error] 8#8: *28057 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/0 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/0", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:15:19 [error] 8#8: *28061 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/0 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/0", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:15:19 [error] 8#8: *28059 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/0 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/0", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:15:22 [error] 9#9: *28068 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:15:22 [error] 8#8: *28063 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/0 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/0", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:15:22 [error] 8#8: *28064 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:15:22 [error] 8#8: *28066 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:15:22 [error] 8#8: *28067 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/counsels HTTP/1.1", upstream: "http://172.19.0.4:8081/api/counsels", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:15:22 [error] 8#8: *28065 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/0 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/0", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:15:25 [error] 8#8: *28075 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:15:25 [error] 8#8: *28076 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:15:25 [error] 8#8: *28079 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:15:31 [error] 8#8: *28081 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 21:31:20 [error] 8#8: *28174 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "GET /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "https://porest.kr/profile"
2021/05/19 21:31:20 [error] 8#8: *28172 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "GET /api/counsels HTTP/1.1", upstream: "http://172.19.0.4:8081/api/counsels", host: "porest.kr", referrer: "https://porest.kr/profile"
2021/05/19 21:31:24 [error] 8#8: *28173 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "PUT /api/profileImg/0 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/0", host: "porest.kr", referrer: "https://porest.kr/main/mainisland"
2021/05/19 23:09:58 [error] 8#8: *29212 connect() failed (113: Host is unreachable) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "OPTIONS /api/email-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/email-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 23:10:02 [error] 8#8: *29208 connect() failed (113: Host is unreachable) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "OPTIONS /api/email-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/email-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 23:10:08 [error] 8#8: *29222 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "OPTIONS /api/email-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/email-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/19 23:10:12 [error] 8#8: *29226 connect() failed (111: Connection refused) while connecting to upstream, client: 116.126.81.104, server: porest.kr, request: "OPTIONS /api/email-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/email-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 06:55:15 [error] 8#8: *31287 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 06:55:19 [error] 8#8: *31297 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:01:49 [error] 8#8: *31392 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/5 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/5", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:01:49 [error] 8#8: *31398 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/5 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/5", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:01:49 [error] 8#8: *31394 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/4 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/4", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:01:49 [error] 8#8: *31400 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/0 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/0", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:01:52 [error] 8#8: *31403 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/0 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/0", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:01:52 [error] 8#8: *31404 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/0 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/0", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:01:52 [error] 8#8: *31402 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/5 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/5", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:01:52 [error] 8#8: *31405 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/0 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/0", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:01:52 [error] 8#8: *31396 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/0 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/0", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:01:55 [error] 8#8: *31412 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/5 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/5", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:01:55 [error] 8#8: *31411 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/0 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/0", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:01:55 [error] 8#8: *31410 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/0 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/0", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:01:55 [error] 8#8: *31413 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/4 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/4", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:01:55 [error] 8#8: *31418 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/3 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/3", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:01:55 [error] 8#8: *31379 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/4 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/4", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:01:58 [error] 9#9: *31421 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/5 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/5", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:01:58 [error] 8#8: *31420 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/4 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/4", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:01:58 [error] 8#8: *31424 connect() failed (113: Host is unreachable) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/profileImg/5 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/profileImg/5", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:05:07 [error] 8#8: *31444 upstream timed out (110: Operation timed out) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/verify-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/verify-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:09:02 [error] 8#8: *31484 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/bans?page=0size=20 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/bans?page=0size=20", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:17:57 [error] 8#8: *31533 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:17:57 [error] 8#8: *31534 connect() failed (111: Connection refused) while connecting to upstream, client: 219.250.207.224, server: porest.kr, request: "OPTIONS /api/counsels HTTP/1.1", upstream: "http://172.19.0.4:8081/api/counsels", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:23:25 [error] 8#8: *31581 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/email-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/email-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:23:25 [error] 8#8: *31583 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/email-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/email-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:23:30 [error] 8#8: *31579 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/email-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/email-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 07:23:33 [error] 8#8: *31585 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/email-check HTTP/1.1", upstream: "http://172.19.0.4:8081/api/email-check", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 11:38:01 [error] 8#8: *32638 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 11:38:04 [error] 8#8: *32636 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 11:38:04 [error] 8#8: *32640 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 11:38:10 [error] 8#8: *32642 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 11:53:28 [error] 8#8: *32737 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 11:53:28 [error] 8#8: *32731 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 11:53:28 [error] 8#8: *32733 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 11:53:31 [error] 8#8: *32735 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 11:53:34 [error] 8#8: *32729 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 11:53:57 [error] 8#8: *32739 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 11:54:04 [error] 8#8: *32741 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 11:54:06 [error] 8#8: *32745 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "GET /api/emotions?month=5&year=2021 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/emotions?month=5&year=2021", host: "porest.kr", referrer: "https://porest.kr/feeling-record"
2021/05/20 13:39:34 [error] 8#8: *33395 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "GET /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 13:39:34 [error] 8#8: *33415 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 13:39:37 [error] 8#8: *33417 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 13:44:49 [error] 8#8: *33463 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/emotions?month=5&year=2021 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/emotions?month=5&year=2021", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 15:53:14 [error] 7#7: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "POST /jenkins/ajaxBuildQueue HTTP/1.1", upstream: "http://172.19.0.5:8080/jenkins/ajaxBuildQueue", host: "porest.kr", referrer: "https://porest.kr/jenkins/"
2021/05/20 15:53:14 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "POST /jenkins/ajaxExecutors HTTP/1.1", upstream: "http://172.19.0.5:8080/jenkins/ajaxExecutors", host: "porest.kr", referrer: "https://porest.kr/jenkins/"
2021/05/20 15:53:17 [error] 7#7: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "POST /jenkins/ajaxBuildQueue HTTP/1.1", upstream: "http://172.19.0.5:8080/jenkins/ajaxBuildQueue", host: "porest.kr", referrer: "https://porest.kr/jenkins/"
2021/05/20 15:53:17 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "POST /jenkins/ajaxExecutors HTTP/1.1", upstream: "http://172.19.0.5:8080/jenkins/ajaxExecutors", host: "porest.kr", referrer: "https://porest.kr/jenkins/"
2021/05/20 17:52:08 [error] 7#7: *1290 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 17:52:13 [error] 7#7: *1292 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 17:52:29 [error] 7#7: *1294 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 17:52:33 [error] 7#7: *1296 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 18:45:51 [error] 7#7: *1618 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 18:45:51 [error] 7#7: *1632 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 18:45:51 [error] 7#7: *1626 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 18:45:52 [error] 7#7: *1634 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 18:45:52 [error] 7#7: *1630 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 18:45:52 [error] 7#7: *1636 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 18:45:53 [error] 7#7: *1638 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 18:46:01 [error] 7#7: *1628 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "POST /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "https://porest.kr/login"
2021/05/20 18:46:17 [error] 7#7: *1616 upstream timed out (110: Operation timed out) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 18:59:32 [error] 7#7: *1738 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 19:13:55 [error] 7#7: *1928 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "POST /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "https://porest.kr/login"
2021/05/20 19:15:15 [error] 7#7: *1955 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "GET /api/counsels/34 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/counsels/34", host: "porest.kr", referrer: "https://porest.kr/mailbox/received"
2021/05/20 19:15:58 [error] 7#7: *1951 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "GET /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "https://porest.kr/mailbox/random-mail"
2021/05/20 19:16:03 [error] 7#7: *1968 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "GET /api/emotions?month=5&year=2021 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/emotions?month=5&year=2021", host: "porest.kr", referrer: "https://porest.kr/feeling-record"
2021/05/20 19:57:14 [error] 7#7: *2406 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "GET /api/stories HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories", host: "porest.kr", referrer: "https://porest.kr/profile"
2021/05/20 19:57:14 [error] 7#7: *2405 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "GET /api/counsels HTTP/1.1", upstream: "http://172.19.0.4:8081/api/counsels", host: "porest.kr", referrer: "https://porest.kr/profile"
2021/05/20 20:04:23 [error] 7#7: *2449 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "POST /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "https://porest.kr/login"
2021/05/20 20:04:23 [error] 7#7: *2447 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "POST /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "https://porest.kr/login"
2021/05/20 20:04:23 [error] 7#7: *2451 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "POST /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "https://porest.kr/login"
2021/05/20 20:04:23 [error] 7#7: *2443 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "POST /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "https://porest.kr/login"
2021/05/20 20:04:23 [error] 7#7: *2445 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "POST /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "https://porest.kr/login"
2021/05/20 20:23:45 [error] 7#7: *2625 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "POST /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "https://porest.kr/login"
2021/05/20 20:23:45 [error] 7#7: *2628 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "POST /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "https://porest.kr/login"
2021/05/20 20:23:48 [error] 7#7: *2634 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "GET /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "https://porest.kr/mailbox/random-mail"
2021/05/20 20:23:48 [error] 7#7: *2634 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "GET /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "https://porest.kr/mailbox/random-mail"
2021/05/20 20:37:12 [error] 7#7: *2729 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 20:37:12 [error] 7#7: *2727 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 20:37:12 [error] 7#7: *2725 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 20:37:19 [error] 7#7: *2723 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 20:37:19 [error] 7#7: *2731 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 20:37:36 [error] 7#7: *2733 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "OPTIONS /api/login HTTP/1.1", upstream: "http://172.19.0.4:8081/api/login", host: "porest.kr", referrer: "http://localhost:8080/"
2021/05/20 21:21:25 [error] 9#9: *2992 connect() failed (111: Connection refused) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "GET /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "https://porest.kr/mailbox/random-mail"
2021/05/20 21:21:47 [error] 8#8: *2982 upstream timed out (110: Operation timed out) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "GET /api/stories/candidates HTTP/1.1", upstream: "http://172.19.0.4:8081/api/stories/candidates", host: "porest.kr", referrer: "https://porest.kr/mailbox/random-mail"
2021/05/20 21:42:03 [error] 7#7: *3090 connect() failed (113: Host is unreachable) while connecting to upstream, client: 49.143.166.68, server: porest.kr, request: "GET /api/emotions?month=5&year=2021 HTTP/1.1", upstream: "http://172.19.0.4:8081/api/emotions?month=5&year=2021", host: "porest.kr", referrer: "https://porest.kr/feeling-record"
